content/help/integrations/wordpress/wishlist.md

```diff
@@ -1,12 +0,0 @@
----
-title: WishList Integration
```

Comment:
Missing redirects for this page, don't forget old redirect.

Reply:
Fixed